react/WishlistAdmin.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-loop-func */
+import React, { FC } from 'react'
+import { injectIntl, defineMessages } from 'react-intl'
+import {
+  Layout,
+  PageBlock,
+  PageHeader,
+  ButtonWithIcon,
+  IconDownload,
+} from 'vtex.styleguide'
+import XLSX from 'xlsx'
+import { useRuntime } from 'vtex.render-runtime'
+
+const WishlistAdmin: FC<any> = ({ intl }) => {
+  const { account } = useRuntime()
+
+  const fetchWishlists = async (from: number, to: number) => {
+    const response = await fetch(
+      `https://${account}.myvtex.com/_v/wishlist/export-lists?from=${from}&to=${to}`,
```

Comment:
You can use relative path for the URL, no need to inform the domain, only `/_v/wishlist/export-lists?from=${from}&to=${to}`